condition3_3.py

```diff
@@ -0,0 +1,13 @@
+"""Write a Python program to print messages on the screen as long as a condition is met.
+"""
+import getch as getch
+
+message =("m1", "m2", "m3")
+
+import keyboard
+import random
+
+while True:  # making a loop
+    print(random.choice(message))
+    if keyboard.is_pressed('space'):  # if key 'q' is pressed
+        break  # finishing the loop
```

Comment:
While running this file, I see displaying continuous values from message list.
Program did not stopped when i pressed space. How can this program stopped?